### ISSUE TITLE
Build errors: Cannot find hooks_c.h

### ISSUE THREAD

**Issue by rohitvarkey** [DynoGraph/stinger-dynograph#1]
Hi,

I was trying to build stinger-dynograph on a Ubuntu 14.04 machine and I get the following error

```shell
In file included from /home/mininet/stinger-dynograph/build/include/stinger_core/stinger.h:500:0,
                 from /home/mininet/stinger-dynograph/build/include/stinger_net/stinger_alg.h:10,
                 from /home/mininet/stinger-dynograph/build/include/stinger_alg/dynamic_betweenness.h:8,
                 from /home/mininet/stinger-dynograph/lib/stinger/src/clients/algorithms/betweenness/src/betweenness.cpp:4:
/home/mininet/stinger-dynograph/build/include/stinger_core/stinger_traversal.h:10:21: fatal error: hooks_c.h: No such file or directory
 #include <hooks_c.h>
                     ^
compilation terminated.
make[2]: *** [lib/stinger/src/clients/algorithms/CMakeFiles/stinger_betweenness.dir/betweenness/src/betweenness.cpp.o] Error 1
make[1]: *** [lib/stinger/src/clients/algorithms/CMakeFiles/stinger_betweenness.dir/all] Error 2
make: *** [all] Error 2
```

I tried a few things to solve this like:

- Adding quotes around the `target_include_directories` paths
- Adding a separate `include_directories("lib/hooks")`

None of these seem to work. Do you have any tips on how I can solve this?

Thanks,
Rohit

**Comment by ehein6** [comment]
The targets that are failing to build are internal to stinger, you don't need them to run the benchmark. To skip building them, type `make dynograph` instead of just `make`.

Your fixes probably didn't work because they need to be added to CMakeLists.txt in each subdirectory of `lib/stinger/src/clients/`. I'm still looking for a clean way to do this.

**Comment by ehein6** [comment]
Fixed in v2.0.0
